source/server/http/admin.cc

```diff
@@ -790,48 +851,59 @@ bool AdminImpl::removeHandler(const std::string& prefix) {
   return false;
 }
 
-void HystrixHandler::updateHystrixRollingWindow(HystrixData* hystrix_data, Server::Instance& server) {
+void HystrixHandler::updateHystrixRollingWindow(HystrixData* hystrix_data,
+    Server::Instance& server) {
   hystrix_data->stats_->incCounter();
 
   for (const Stats::CounterSharedPtr& counter : server.stats().counters()) {
+    // we save all upstream_rq stats. could be more specific.
```

Comment:
Didn't understand the comment. I think we should delete it

Reply:
ok